### ISSUE TITLE
Only two status codes

### ISSUE THREAD

**Issue by masciyh** [jeroenterheerdt/python-egardia#21]
I am running the latest version of Hassbian, and trying to connect to my WV-1716.
Basic installation runs alright, I get the proper armed/armed away/disarmed status in home assistant.

Now I changed my WV-1716 settings to report to home assistant. I uploaded the python-egardia files and am able run the egardiaserver.py script using putty. The script does show some status codes, but only one for status armed and one for disarmed. The one for armed and armed away is the same one, and also the same one if I use any of my two keypads. And more important, I don't get a code when the alarm is triggered. In 'history records' in my alarm panel I do see the alarm is registered correctly. I just don't get the codes using the script. Am I doing something wrong here?

**Comment by jeroenterheerdt** [comment]
in order for the trigger codes to be sent you need to change the config of the alarm itself. Did you follow this (specifically step 2): https://www.home-assistant.io/components/egardia#advanced-configuration?

**Comment by masciyh** [comment]
Yes, I changed the server to 192.168.1.108 which is the address of my raspberry pi. I also tried different port numbers (the standard one, but also some others, also changing it in the command line of the script). The script does send some codes, but only a few.

**Comment by jeroenterheerdt** [comment]
ok, that is weird. can you verify that the alarm system is actually sending these codes by using a network sniffer?

**Comment by masciyh** [comment]
I am not sure how to do that? Logged into the alarm panel, I can see all changes coming true, so the alarm is being triggered and registered. You want me to check if there is actually traffic sent to the IP I mentioned?

**Comment by jeroenterheerdt** [comment]
you can check that, but also just use something like wireshark and log any traffic from the alarm panel itself to verify that the alarm does indeed send anything.

**Comment by masciyh** [comment]
I installed wireshark and can see there is traffic from my alarm panel (192.168.1.106). I am not sure where it is going, it shows the IP of my laptop as destination, not the one of my PI (.108).

**Comment by masciyh** [comment]
I think it was just me checking the alarm status in the alarm panel (.106). I tried again, without checking the control panel in the browser. In that case, I don't see any traffic coming from .106 (the panel). Not when arming/disarming and not when triggering an alarm. Under System Settings, Report, Center Server, I just typed the IP of the the raspberry (.108). That should be correct, right?

**Comment by jeroenterheerdt** [comment]
That should have been enough yes.

**Comment by Knol010** [comment]
@jeroenterheerdt is it possible to execute "sudo python3 egardiaserver.py" on Hass.io standard installation or do i need Hassbian?

**Comment by jeroenterheerdt** [comment]
@Knol010 you need Python3 and some packages. No actual dependency on Home Assistant.

**Comment by SneakSnackSnake** [comment]
I have the WV-1716 and I also have this issue. I would get the same code over and over again.
I wanted to figure out why this is happening. Then I tried to use test_egardiaserver.py to report to ics.wv.1716.alt.egardia.com and I got a specific unicode response: \u0006
I changed the DEFAULT_RESPONSE in egardiaserver.py to: DEFAULT_RESPONSE = '\u0006'
Now I get different codes for each trigger. 
I don't know about GATE-01, GATE-02 and GATE-03, but it seems that the WV-1716 needs the ACKNOWLEDGE unicode character as a response, otherwise it keeps sending the same code.
